configure.ac

```diff
@@ -326,11 +326,6 @@ fi
 AM_CONDITIONAL([WITH_GNUTLS], [test "x$with_gnutls" = "xyes"])
 AC_SUBST([GNUTLS_LIBS])
 
-AC_PATH_PROG([EXPECT], expect)
```

Comment:
Expect is still used here: https://github.com/stefanberger/swtpm/blob/master/samples/swtpm-create-tpmca#L61

Reply:
Good catch, though that's "just" an example (though it is indeed installed to `/usr/share/swtpm`, but not in the default path). Do you think it is worth having a hard failure in this situation?

Reply:
It's run as a test case as well. I think this should fail hard if it's not installed.

Reply:
OK, I omitted the commit removing the check for expect and signed off the other commit.